tests/models/test_marrmotm01.py

```diff
@@ -47,12 +27,27 @@ def forcing_file(self):
         return Path(__file__).parent / 'data' / 'BMI_testcase_m01_BuffaloRiver_TN_USA.mat'
 
     @pytest.fixture
-    def model_with_setup(self, mocked_config, model: MarrmotM01, forcing_file: Path):
-        cfg_file, cfg_dir = model.setup(
-            forcing=forcing_file
-        )
+    def model(self, forcing_file: Path, mocked_config):
+        m = MarrmotM01(version="2020.11", forcing=forcing_file)
+        yield m
+        if m.bmi:
+            # Clean up container
+            del m.bmi
+
+    @pytest.fixture
+    def model_with_setup(self, model: MarrmotM01):
+        cfg_file, cfg_dir = model.setup()
         return model, cfg_file, cfg_dir
 
+    def test_parameters(self, model, forcing_file):
```

Comment:
Are start_time and end_time considered parameters as well? Since they're part of the `setup` function, I'm inclined to see them as such.

Reply:
it is fixed.